src/platforms/apple/common/performance/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: SwiftUI Instrumentation
+sidebar_order: 12
+description: "Learn how to monitor the performance of your SwiftUI views."
+---
+
+<Note>
+    This is an experimental feature.
+</Note>
+
+
+You can monitor the performance of your views in a SwiftUI project with SentrySwiftUI library.
+
+Install it with CocoaPod:
```

Comment:
`m`: I think we should point out that SwiftUI needs an extra package. If I remember correctly, SwiftUI also includes Sentry, so we should clarify that we don't need both.

Maybe it is better to add this info to each of the pages in https://docs.sentry.io/platforms/apple/guides/ios/install/. So basically the code samples there would mention that there is also a package for SwiftUI.

```
platform :ios, '9.0'
use_frameworks! # This is important

target 'YourApp' do
  pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '7.31.5'
  # If you want SwiftUI or something like that
  pod 'SentrySwiftUI', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '7.31.5'
end
```